presto-main-base/src/main/java/com/facebook/presto/server/InternalCommunicationConfig.java

```diff
@@ -187,6 +189,32 @@ public InternalCommunicationConfig setThriftTransportEnabled(boolean thriftTrans
         return this;
     }
 
+    public boolean isTaskUpdateRequestThriftTransportEnabled()
```

Comment:
this flag controls the codec for serde ...and not really the transport right ? transport is still http even after setting this to true right ? name is confusing.